### ISSUE TITLE
how to get session uuid ?

### ISSUE THREAD

**Issue by herbiel** [shimaore/esl#56]
var call_handler = async function() {
  const uuid = @data['Channel-Unique-ID'];
  res = await this.command("playback",'music.wav');
  var foo = res.body.variable_foo;
  console.log(uuid);
};

require('esl').server(call_handler).listen(7000);

**Comment by shimaore** [comment]
I'm in the progress of updating the package and the documentation (this will take some time) but does it work better if you use `this.data` instead of `@data`?

Something along the lines of
```
const call_handler = async function() {
  const uuid = this.data['Channel-Unique-ID'];
  console.log(uuid);
};
require('esl').server(call_handler).listen(7000);
```

Note that `this.uuid` will contain the value of the `Unique-ID` field and might be the value you are looking for.

**Comment by herbiel** [comment]
yes ,i have get uuid ,but now i want to get event ,but still can't ,can you help me ?

this is my event 
Event-Subclass: google_transcribe%3A%3Atranscription
Event-Name: CUSTOM
Core-UUID: 0a872b4e-f2b4-4096-90db-57782afa4e39
FreeSWITCH-Hostname: debian-s-2vcpu-4gb-sgp1-01
FreeSWITCH-Switchname: debian-s-2vcpu-4gb-sgp1-01
FreeSWITCH-IPv4: 165.232.161.156
FreeSWITCH-IPv6: %3A%3A1
Event-Date-Local: 2021-05-29%2002%3A00%3A24
Event-Date-GMT: Sat,%2029%20May%202021%2002%3A00%3A24%20GMT
Event-Date-Timestamp: 1622253624357589
Event-Calling-File: mod_google_transcribe.c
Event-Calling-Function: responseHandler
Event-Calling-Line-Number: 67
Event-Sequence: 19561
Channel-State: CS_EXECUTE
Channel-Call-State: EARLY
Channel-State-Number: 4
Channel-Name: sofia/internal/1002%40165.232.161.156%3A13556
Unique-ID: 697b424a-28c4-47aa-9163-98e4b019cec8
Call-Direction: inbound
Presence-Call-Direction: inbound
Channel-HIT-Dialplan: true
Channel-Presence-ID: 1002%40165.232.161.156
Channel-Call-UUID: 697b424a-28c4-47aa-9163-98e4b019cec8
Answer-State: early
Channel-Read-Codec-Name: PCMA



and my code is like 
  this.on('google_transcribe::transcription',function(o) {
       console.log(o)
  })

but can't get o

**Comment by shimaore** [comment]
The event name is 'CUSTOM', so `this.on('CUSTOM', function({body}) { … })`. You'll have to route based on `body['Event-Subclass']` in this case.
